### ISSUE TITLE
The print preview does not load

### ISSUE THREAD

**Issue by fvanheeswijk** [MicrosoftEdge/WebView2Feedback#4371]
### What happened?

When using the print functionality with the beta runtime the print preview does not load, a loading circle will be shown indefinitely.

### Importance

Blocking. My app's basic functions are not working due to this issue.

### Runtime Channel

Prerelease (Edge Canary/Dev/Beta)

### Runtime Version

122.0.2365.30 beta

### SDK Version

1.0.2277.86

### Framework

Winforms

### Operating System

Windows 11

### OS Version

10.0.0.22631

### Repro steps

With the Microsoft Edge Beta application installed the following code should be run in a C# Windows Forms application. Subsequently the Print command should be selected in the context menu after opening the menu by right-clicking on the page.

```
using System;
using System.Drawing;
using System.Windows.Forms;

using Microsoft.Web.WebView2.WinForms;

namespace WebView2Bug
{
    static class Program
    {
        [STAThread]
        static void Main()
        {
            Environment.SetEnvironmentVariable("WEBVIEW2_RELEASE_CHANNEL_PREFERENCE", "1");

            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(CreateForm());
        }

        private static Form CreateForm()
        {
            var form = new Form
            {
                Size = new Size(1280, 1024)
            };

            var webView2 = new WebView2
            {
                Dock = DockStyle.Fill,
                Source = new Uri("http://example.org/")
            };

            form.Controls.Add(webView2);

            return form;
        }
    }
}
```

### Repros in Edge Browser

No

### Regression

Regression in newer Runtime

### Last working version (if regression)

121.0.2277.112

[AB#48981284](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/48981284)

**Comment by JosephJin0815** [comment]
I am able to repro this on webview2. We are tracking it in our internal backlog.

**Comment by monica-ch** [comment]
@fvanheeswijk The issue is fixed in Edge canary 123.0.2407.0+. Please validate and let us know if you have any questions!

**Comment by fvanheeswijk** [comment]
When can we expect the fix in Beta?

**Comment by monica-ch** [comment]
The fix is backported and should be available in Edge beta 122.0.2365.38+
